apps/web/layouts/default.vue

```diff
@@ -1,10 +1,7 @@
 <template>
   <div>
     <UiHeader />
-    <NarrowContainer v-if="breadcrumbs?.length" class="p-4 md:px-0">
-      <LazyUiBreadcrumbs :breadcrumbs="breadcrumbs" />
-    </NarrowContainer>
-    <main>
+    <main class="min-h-screen">
```

Comment:
We have to figure out why this was there in the first place. Why did we need min-height: 100vh on `<main>`

Reply:
left a comment in card, i can provide more context if you have time.pm.